logs/logging_function.py

```diff
@@ -0,0 +1,6 @@
+import logging
```

Comment:
`configure_logger` or just `logging` looks like more appropriate names for this file